### ISSUE TITLE
Document how to setup usim and chaosnet-bridge 

### ISSUE THREAD

**Issue by ams** [LM-3/usim#8]

**Comment by bictorv** [comment]
What needs to be improved in the instructions for chaosnet-bridge? More examples?

**Comment by ams** [comment]
Yeah, I was thinking more examples, and putting one or two into the `usim/README`, and maybe some links to _chaosnet-bridge_.

**Comment by ams** [comment]
Managed to get usim and cbridge to play along:

![cadr-fun](https://user-images.githubusercontent.com/7945138/48164036-aa734480-e2e1-11e8-9f12-bf022b45d806.png)

The following should need to be fixed:

https://github.com/LM-3/usim/issues/20
https://github.com/LM-3/chaos/issues/60
https://github.com/LM-3/chaos/issues/59

**Comment by ams** [comment]
This should be done with #13. So that the instructions are complete from compiling usim to getting it in the Global Chaosnet.
